README.md

```diff
@@ -1,198 +1,163 @@
 
-_Examples_
+#### options
 
-```js
-client.js; // [{ type: 'module', value: 'http://<asset server>/finn/js/my-app/1.0.0/index.js' }]
-```
+| option      | default         | type       | required | details                                                                          |
+| ----------- | --------------- | ---------- | -------- | -------------------------------------------------------------------------------- |
+| maps        | `false`         | `boolean`  | `false`  | Specifies whether import maps defined in the config should be loaded from the Eik server or not. |
 
-```js
-client.js[0].toHTML(); // <script type="module" src="http://<asset server>/finn/js/my-app/1.0.0/index.js">
-```
 
-### client.css
+### .files(file)
 
-Returns an array of CSS asset objects for the given mode (development or non development) based on values in `assets.json`
-As asset object can be serialized using `JSON.stringify` or converted into an HTML link tag using the method `.toHTML()`
+Constructs a full URL to an asset. The URL is built up by appending the value of the `file` argument to a `base` root. By default (production mode) the `base` root is built up from values in Eik config matching where the package for the config are located on the Eik server. If the module is in development mode, the value set for `base` on the constructor will be used as the `base` root.
 
-_Example_
+#### arguments
 
-```js
-client.css; // [{ type: 'default', value: 'http://<asset server>/finn/css/my-app/1.0.0/index.css' }]
-```
+| option      | default         | type       | required | details                                                                          |
+| ----------- | --------------- | ---------- | -------- | -------------------------------------------------------------------------------- |
+| file        | `null`          | `string`   | `false`  | File to append to the base of a full URL to an asset                             |
+
+Returns a object with as follow:
 
 ```js
-client.css[0].toHTML(); // <link type="text/stylesheet" rel="stylesheet" href="http://<asset server>/finn/css/my-app/1.0.0/index.css">
+{
+  integrity: 'sha512-zHQjnDpMW7IKVyTpT9cOPT1+xhUSOcbgXj6qHCPSPu1CbQfgwDEsIniXU54zDIN71zqmxLSp3hfIljpt69ok0w==',
+  value: 'https://cdn.eik.dev/pkg/mymodue/2.4.1/path/script.js'
+}
 ```
 
-### client.scripts
+If `integrity` of the file is not available, the value for `integrity` will be `null`. This will be the case when in development mode since integrity is calculated upon publish of a package to a Eik server.
 
-Returns JavaScript script tag markup for the given mode (development or non development) based on values in `assets.json`
+### .maps()
 
-_Examples_
+Returns the image maps defined in Eik config from the Eik server. For the maps to be returned they need to be loaded from the Eik server. This is done by setting the `maps` option on the `.load()` method to `true`.
```

Comment:
`image maps`? Not sure what is meant by this.

Reply:
Spelling error. That should be `import map`.